demo.py

```diff
@@ -88,8 +88,8 @@ def print_sep(char='-'):
         print 'Sending authentication request...'
         
         try:
-            request_status = api.authenticate(pairing_id, terminal_name, terminal_name_extra=terminal_extra)
-            request_id = request_status.id
+            auth_request_status = api.authenticate(pairing.id, terminal_name, terminal_name_extra=terminal_extra)
```

Comment:
Does this work? We initialize a variable called `terminal_extras` but use `terminal_extra` here.